src/main/sensors/barometer.c

```diff
@@ -40,6 +40,7 @@
 #include "drivers/barometer/barometer_fake.h"
 #include "drivers/barometer/barometer_ms5611.h"
 #include "drivers/barometer/barometer_lps.h"
+#include "drivers/barometer/barometer_dps310.h"
```

Comment:
the extension is included but returning an error

Reply:
you need to add some code to the makefile to ensure that the barometer_dps310.c file is being compiled.

Reply:
thanks, the compilation format of EmuFlight, BetaFlight and iNav is still pretty confusing to me